launchable/plugin.py

```diff
@@ -92,7 +92,8 @@ def afterTest(self, test):
 
     @protect
     def addError(self, test, err, capt=None):
-        self._addResult(test, CaseEvent.TEAT_FAILED, self._uploader.enqueue_failure)
+        if err is not ImportError:
```

Comment:
It seems if a non-existing test is specified, err is tuple like: `(<class 'ValueError'>, ValueError('No such test aaa'), <traceback object at 0x1044a2040>)`. You can check this by running `nosetests tests/dir3/test1.py:aaa -v --nocapture --nologcapture --launchable --launchable-build-number 1` in the rocket-car repo!

Reply:
I fixed to handle ValueError type